templates/settings.php

```diff
@@ -38,12 +38,21 @@
 							</p>
 							<p>
 								<label>
-									<input type="checkbox" name="activitypub_enable_blog_user" id="activitypub_enable_blog_user" value="1" <?php echo \checked( '1', \get_option( 'activitypub_enable_blog_user', '0' ) ); ?> />
-									<?php \esc_html_e( 'Enable Blog-Profile', 'activitypub' ); ?>
+									<input type="radio" name="activitypub_actor_mode" id="activitypub_actor_mode" value="2" <?php echo \checked( '2', \get_option( 'activitypub_actor_mode', '1' ) ); ?> />
+									<strong><?php \esc_html_e( 'Only Blog-Profile', 'activitypub' ); ?></strong>
 								</label>
 							</p>
 							<p class="description">
-								<?php \esc_html_e( 'Your blog becomes an ActivityPub profile.', 'activitypub' ); ?>
+								<?php \esc_html_e( 'Your blog becomes an ActivityPub profile and every post will be published under this profile instead of the author profile.', 'activitypub' ); ?>
```

Comment:
```suggestion
								<?php \esc_html_e( 'Your blog becomes a single ActivityPub profile and every post will be published under this profile instead of the individual author profiles.', 'activitypub' ); ?>
```